www/artists/mbolen/gallery.html

```diff
@@ -0,0 +1,110 @@
+    <!DOCTYPE html>
+    <html lang="en">
+        <head>
+            <title>Mel Bolen's Gallery</title>
+            <meta charset="utf-8">
+            <meta http-equiv="X-UA-Compatible" content="IE=edge">
+            <meta name="viewport" content="width=device-width, initial-scale=1">
+            <meta name="description" content="Portfolio for Connor Brezinsky">
+            <meta name="author" content="Connor Brezinsky">
+            <!-- Bootstrap Core CSS -->
+            <link href="css/bootstrap.min.css" rel="stylesheet"><!-- Custom CSS -->
+            <link href="css/stylish-portfolio.css" rel="stylesheet"><!-- Custom Fonts -->
+            <link href="font-awesome/css/font-awesome.min.css" rel="stylesheet" type="text/css">
+            <link href="css/onepage-scroll.css" rel="stylesheet" type="text/css">
+            <link href="css/styles.css" rel="stylesheet">
+			<link href="css/galleryslideshow.css" rel="stylesheet"><!-- For slideshow -->
+            <link href="http://fonts.googleapis.com/css?family=Source+Sans+Pro:300,400,700,300italic,400italic,700italic" rel="stylesheet" type="text/css">
+            <style type="text/css">
+            div.c9 {margin-top:10em;}
+            a.c8 { color: #050504;}
+            div.c7 {margin-top:8em;}
+            p.c6 {font-size:3em;padding-bottom:1em;}
+            p.c5 {font-family: 'Raleway', sans-serif; font-weight: 300;}
+            h1.c4 { padding-right:4em; font-family: 'Raleway', sans-serif; font-weight: 100;}
+            a.c3 { color: #ffffff;}
+            h1.c2 {font-family: 'Raleway', sans-serif; font-weight: 100;}
+            li.c1 {list-style: none; display: inline}
+            </style>
+	</head>
+	<body>
+	</body>
+               <section>
```

Comment:
The reason this is still giving an error is because it's currently not in the body. You need to fix that. You can do that by removing the </body> that occurs right before this that closes the body with nothing in it.